webapp/channels/src/components/about_build_modal/about_build_modal.test.tsx

```diff
@@ -51,17 +59,22 @@ describe('components/AboutBuildModal', () => {
             IsLicensed: 'true',
             Company: 'Mattermost Inc',
         };
+        socketStatus = {
+            connected: true,
+            serverHostname: 'mock.localhost',
+        };
     });
 
     test('should match snapshot for enterprise edition', () => {
```

Comment:
I piggy-backed off these existing snapshot tests to verify the new behavior. Let me know if this is okay - if not I can isolate the changes out into their own tests :slightly_smiling_face:

Reply:
Yep that's fine! I think we also need to add a test here if `serverHostname` is undefined?  If `server_hostname` isn't in the response then I'm pretty sure the current logic will set it to `undefined` instead of `''`? I noticed the `createHelloMessage()` in `web_conn.go` doesn't include `server_hostname` if `os.Hostname()` throws an error so we just need to ensure we aren't throwing a console error if that case happens

Reply:
I think that should be tested by the case below, right?

https://github.com/mattermost/mattermost/blob/c098997eee69d4dba088d7e852ead871dddce399/webapp/channels/src/components/about_build_modal/about_build_modal.test.tsx#L140-L149

The `socketStatus` object should have an `undefined` server hostname set. Or maybe I'm missing something? I could set the value of `serverHostname` to `undefined` explicitly, too, if that helps with readability :slightly_smiling_face:

Reply:
Ah yes, I missed that one. Thanks! That'll cover it